src/helpers/Day.js

```diff
@@ -0,0 +1,6 @@
+const Day = (function() {
+    const day = new Date();
+    return day.getFullYear() + '/' + (day.getMonth()+1) + '/' + day.getDate();
+})();
+
+export default Day;
```

Comment:
Very good, that you moved this function to helpers!
The name of the function usually describe what this function do, so better name here will be `getCurrentDay` and yeah, if function is not a Class or Component it usually starts from the lower case.
And interesting pattern - why do you use IFFE here? i guess for reason, but it is better to explain it in description to the function